eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -47,17 +47,17 @@ jobs:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
         - (Ubuntu.1804.Arm32.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440
       - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-        - (Debian.9.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-bfcd90a-20200121150037
+        - (Debian.11.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm32v7-20210304164347-5a7c380
```

Comment:
Shouldn't we upgrade Debian 9 to Debian 10 as that's the min supported version?

Reply:
In libraries we have Debian 9 and Debian 10, and coreclr it's just Debian 9. I have only replaced 9. If we don't want multiple Debian queues for coreclr as well, I can change it to 10 as min version?

Reply:
Let's have both Debian 10 and 11 everywhere (in accordance to our support matrix). If that consumes to much resources we can comment them out at any point.